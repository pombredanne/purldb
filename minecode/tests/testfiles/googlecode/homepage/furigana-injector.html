

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "projectName": "furigana-injector", "projectHomeUrl": "/p/furigana-injector", "relativeBaseUrl": "", "loggedInUserEmail": null, "profileUrl": null, "token": null, "domainName": null, "assetHostPath": "http://www.gstatic.com/codesite/ph"};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 _gaq.push(
 ['projectTracker._setAccount', 'UA-25131337-2'],
 ['projectTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 furigana-injector -
 
 
 Inserts furigana for kanji words (Mozilla Firefox and Google Chrome extensions) - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Ffurigana-injector%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Ffurigana-injector%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/furigana-injector">
 <a href="/p/furigana-injector/">
 
 
 <img src="/p/furigana-injector/logo?cct=1314582832"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/furigana-injector/"><span itemprop="name">furigana-injector</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/furigana-injector/"><span itemprop="description">Inserts furigana for kanji words (Mozilla Firefox and Google Chrome extensions)</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/furigana-injector/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/furigana-injector/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/furigana-injector/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/furigana-injector/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/furigana-injector/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/furigana-injector/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/furigana-injector/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/furigana-injector/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://www.gnu.org/licenses/old-licenses/gpl-2.0.html" rel="nofollow">GNU GPL v2</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:furigana">furigana</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:mozilla">mozilla</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:xpcom">xpcom</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:nihongo">nihongo</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:振りがな">振りがな</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:フリガナ">フリガナ</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:ふりがな">ふりがな</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/akira.kurogane@gmail.com/">akira.kurogane@gmail.com</a
>
 
 
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 <div class="phead pfeatured">Featured</div>
 
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_paper_pencil-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Wiki pages</b></span>
 </div>
 <ul class="pslist nowrap">
 
 <li class="psmeta">
 <a href="wiki/FuriganaInjectorOriginalMotivation" title="The original motivation for developing Furigana Injector"
 >FuriganaInjectorOriginalMotivation</a>
 </li>
 
 <li class="psmeta">
 <a href="wiki/LeaveAComment" title="A placeholder wiki document for general comments"
 >LeaveAComment</a>
 </li>
 
 <li class="psmeta"><a href="w/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 
 <li class="psmeta"><b>External links</b></li>
 
 <li class="psmeta">
 <a href="https://addons.mozilla.org/en-US/firefox/addon/6178" rel="nofollow">Official Firefox download</a>
 </li>
 
 <li class="psmeta">
 <a href="https://chrome.google.com/extensions/detail/cbahnmcliajmanjkaolemjelphicnein" rel="nofollow">Official Chrome download</a>
 </li>
 
 <li class="psmeta">
 <a href="http://sourceforge.jp/magazine/09/05/27/1919234" rel="nofollow">漢字に自動的にルビを振ってくれる</a>
 </li>
 
 <li class="psgap"/>
 
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <p><i>ふりがなインジェクター - Nanoszenie furigany - 振假名植入器 - حاقن الفوريغانا </i> </p><h2><a name="Furigana_Injector_has_been_shutdown_(as_of_Dec_31_2013)"></a>Furigana Injector has been shutdown (as of Dec 31 2013)<a href="#Furigana_Injector_has_been_shutdown_(as_of_Dec_31_2013)" class="section_anchor"></a></h2><p>Feel free to fork any code here if you&#x27;d like to make a replacement. </p><p>udonmai-san has built the furigana*server<strong> on MacOSX, so that he can run it all locally on his desktop/laptop.  </strong></p><p>His github of his own version is at <a href="https://github.com/udonmai/furigana-injector/" rel="nofollow">https://github.com/udonmai/furigana-injector/</a>. The instructions of his changes are on the following wiki page: <a href="/p/furigana-injector/wiki/Udonmai_MacOSX_build">Udonmai_MacOSX_build</a>. </p><h2><a name="Furigana_Injector_was_..."></a>Furigana Injector was ...<a href="#Furigana_Injector_was_..." class="section_anchor"></a></h2><blockquote>the sum of these parts: 
</blockquote><ul><li><a href="http://mecab.sourceforge.net/" rel="nofollow">MeCab</a> natural-language parser (server or XPCOM component.) </li><li>Adjustable array of user&#x27;s known kanji (javascript) </li><li>Webpage-text parsing (javascript) </li><li>Webpage DOM manipulation to insert <tt>&lt;RUBY&gt;</tt> tags. (javascript) </li><li>GUI controls and help pages (javascript, image and html resources) </li></ul><p>If you want to contact the developer you may freely email him at <strong>akira.kurogane</strong> at gmail. If you want to use the server program as a localhost service you will have to compile for yourself, so <strong>experienced C++ project builders only</strong> thanks. </p><h2><a name="Localization"></a>Localization<a href="#Localization" class="section_anchor"></a></h2><blockquote>Thanks to Erii-san (Italy), Philippe-san (France), Daniel (Germany), Konrad (Poland), Andre (Brazil), Ahmad (Lebanon), Kayo (Japan), Ji-Hyuk (Korea), CC Fan (Chinese), Nesukun (Spain) and Carmen (Dutch) for their localizations! 
</blockquote>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

