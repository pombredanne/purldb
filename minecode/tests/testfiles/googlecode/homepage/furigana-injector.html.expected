{'content_license': None,
 'content_license_url': None,
 'description': u'Inserts furigana for kanji words (Mozilla Firefox and Google Chrome extensions)',
 'downloads_url': u'http://code.google.com/p/furigana-injector/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/furigana-injector/feeds',
 'home_url': u'http://code.google.com/p/furigana-injector/',
 'id': u'furigana-injector',
 'issues_url': u'http://code.google.com/p/furigana-injector/issues/list',
 'keywords': [],
 'labels': [u'furigana',
            u'mozilla',
            u'nihongo',
            u'xpcom',
            u'\u3075\u308a\u304c\u306a',
            u'\u30d5\u30ea\u30ac\u30ca',
            u'\u632f\u308a\u304c\u306a'],
 'license': u'GNU GPL v2',
 'license_url': u'http://www.gnu.org/licenses/old-licenses/gpl-2.0.html',
 'name': u'furigana-injector',
 'peoples_url': u'http://code.google.com/p/furigana-injector/people/list',
 'vcs_url': u'http://code.google.com/p/furigana-injector/source/checkout',
 'wiki_url': u'http://code.google.com/p/furigana-injector/w/list'}